### DISCUSSION TITLE
ROCm Version 5.2 supported NAVI 10 devices RX5500 XT and RX6750XT. Version 5.7 and current does not. Why

### DISCUSSION THREAD

**Discussion by housebyte** [ROCm/ROCm#2629] [General]
I have ROCm 5.2 setup on my RedHat Machine and am running many AI projects. I have upgraded my Ubuntu Debian based OS to Jammy because I now have an AMD Ryzen 6-core cpu. And I find out that ROCm (AMD) has now ensured that NAVI 10 cards are not in anyway supported. Rather than unsupported and needing the simple fix 
`export HSA_OVERRIDE_GFX_VERSION=10.3.0`
This is unaccacceptable. 
Ubuntu Jammy can use down to version 5.3 - I tried version 5.3 and this also doesnt work for my card. 5.2 is not supported on Jammy. I really think all NAVI 10 cards should be supported.
I paid good money for my GPU's and I am not willing to pay double that for a supported card.
Fix it please.

(Well done AMD for everything else)

**Comment by johnnynunez** [comment]
> I have ROCm 5.2 setup on my RedHat Machine and am running many AI projects. I have upgraded my Ubuntu Debian based OS to Jammy because I now have an AMD Ryzen 6-core cpu. And I find out that ROCm (AMD) has now ensured that NAVI 10 cards are not in anyway supported. Rather than unsupported and needing the simple fix `export HSA_OVERRIDE_GFX_VERSION=10.3.0` This is unaccacceptable. Ubuntu Jammy can use down to version 5.3 - I tried version 5.3 and this also doesnt work for my card. 5.2 is not supported on Jammy. I really think all NAVI 10 cards should be supported. I paid good money for my GPU's and I am not willing to pay double that for a supported card. Fix it please.
> 
> (Well done AMD for everything else)

Upgrade to rocm5.7. Use pytorch nightly or compile your tensorflow. 

use prepare.sh with sudo and then build tensorflow
https://github.com/johnnynunez/rocm_lab/blob/master/scripts/build_tensorflow.sh
https://hackmd.io/YKSIbRLfQS6VWWrgrHwU9g
https://github.com/RadeonOpenCompute/ROCm/issues/1880#issuecomment-1792463773
